crates/tauri-plugin-holochain-service-consumer/rollup.config.js

```diff
@@ -15,11 +15,6 @@ export default [
         file: "dist-js/holochain-env/index.min.js",
         format: 'esm',
       },
-      {
-        // Android resource filenames must be lowercase and have only a single '.' extension
-        file: "android/src/main/res/raw/holochainenv.js",
```

Comment:
We're no longer injecting the `__HC_LAUNCHER_ENV__` from the kotlin class, so we don't need this included in the android resources.